scripts/update-dists-own-components-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$TAGPR_CURRENT_VERSION" ]; then
+  echo "Error: TAGPR_CURRENT_VERSION environment variable is not set"
+  exit 1
+fi
+
+if [ -z "$TAGPR_NEXT_VERSION" ]; then
+  echo "Error: TAGPR_NEXT_VERSION environment variable is not set"
```

Comment:
maybe it is better to output to stderr. `>&2`

Reply:
fixed at 88e6b86